package.json

```diff
@@ -6,22 +6,22 @@
   },
   "exports": {
     ".": {
-      "node": "./lib.cjs/index.js",
-      "default": "./lib.esm/index.js"
+      "require": "./lib.cjs/index.cjs",
+      "import": "./lib.esm/index.mjs"
```

Comment:
Used Michael JackSon (`.mjs`) file extension to forcefully push node into ESM mode.